### DISCUSSION TITLE
[Feature request] Sidebar for boosts etc.

### DISCUSSION THREAD

**Discussion by Tazka** [runelite/runelite#15057] [Ideas]
Boosts and other information are shown on the game-screen as an overlay at the moment. They can be rearranged, which is fine for resizeable users, but fixed screen users run out of screen space quite quickly. It would be great if stat boosts and other information (e.g. slayer task, monster hp) could be displayed on a sidebar as a toggleable option.

**Comment by edddyee** [comment]
Wonderful idea. I like to play with fixed screen too, and sometimes the stats boots and other information take too much space.
